lib/primitives/array-element.js

```diff
@@ -38,6 +38,18 @@ module.exports = function(BaseElement, registry) {
       return [this.element, this.meta.toValue(), attributes, compactDoms];
     },
 
+    toEmbeddedRefract: function() {
+      if (this.attributes.content.length === 0 && this.meta.content.length === 0) {
```

Comment:
Been wanting to add this for a while. Will do on a later PR.

https://github.com/refractproject/minim/issues/57